apiary.apib

```diff
@@ -1706,22 +1706,802 @@ root will not be listable, as it is virtual. Access known paths directly.
++ Response 200
+
+    + Headers
+
+        ```
+        ```
+
+    + Attributes (object)
+        - Strings (array, nullable)
+            - "/ip4/192.168.0.0/ipcidr/16" (MultiAddr)
+
+    + Body
+
+        ```
+        {
+            "Strings": null
+        }
+        ```
+
+
+## filters add [GET /swarm/filters/add{?arg}]
+Add the given multiaddr to the filter list.
+
+'ipfs swarm filters add' will add an address filter to the daemon's swarm.
+Filters applied this way will not persist daemon reboots, to acheive that,
+add your filters to the ipfs config file.
+
++ Parameters
+    + arg (MultiAddr, required) - Address to add to the filter list
+
++ Request Without Arguments
+
+    #### curl
+
+        curl -i "http://localhost:5001/api/v0/swarm/filters/add"
+
+    + Body
+
+        ```
+        curl -i "http://localhost:5001/api/v0/swarm/filters/add"
+        ```
+
++ Response 400
+
+    + Headers
+
+        ```
+        Date: Thu, 11 Feb 2016 23:17:09 GMT
+        Content-Length: 30
+        Content-Type: text/plain; charset=utf-8
+        ```
+
+    + Attributes (string)
+
+    + Body
+
+        ```
+        Argument 'address' is required
+        ```
+
++ Request With Empty Argument
+
+    The response is the same if the argument is invalid. For example:
+
+        curl -i "http://localhost:5001/api/v0/swarm/filters/add?arg=kitten"
+
+    #### curl
+
+        curl -i "http://localhost:5001/api/v0/swarm/filters/add?arg="
+
+    + Body
+
+        ```
+        curl -i "http://localhost:5001/api/v0/swarm/filters/add?arg="
+        ```
+
++ Response 500
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Output, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Output, X-Chunked-Output
+        Content-Type: application/json
+        Server: go-ipfs/0.4.0-dev
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Thu, 11 Feb 2016 23:17:31 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (Error)
+        - Message: "invalid multiaddr-filter format"
+        - Code: 0
+
+    + Body
+
+        ```
+        {
+          "Message": "invalid multiaddr-filter format",
+          "Code": 0
+        }
+        ```
+
++ Request With Argument
+
+    #### curl
+
+        curl -i "http://localhost:5001/api/v0/swarm/filters/add?arg=/ip4/192.168.0.0/ipcidr/16"
+
+    + Body
+
+        ```
+        curl -i "http://localhost:5001/api/v0/swarm/filters/add?arg=/ip4/192.168.0.0/ipcidr/16"
+        ```
+
++ Response 200
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Output, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Output, X-Chunked-Output
+        Content-Type: application/json
+        Server: go-ipfs/0.4.0-dev
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Thu, 11 Feb 2016 23:18:50 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (string)
+
+    + Body
```

Comment:
From the blueprint docs

> All of the blueprint sections are optional.

So I suggest filling out only sections that actual contain content. But if you want to discuss this further we can pull this into an individual issue.